packages/plugin-awesome/test/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Please remove the file as well. No need to provide additional typescript configuration there